src/core/src/op/paged_attention.cpp

```diff
@@ -19,8 +19,8 @@ void PagedAttentionExtension::validate_and_infer_types() {
     OV_OP_SCOPE(PagedAttentionExtension_validate_and_infer_types);
 
     NODE_VALIDATION_CHECK(this,
-                          get_input_size() == 13,
-                          "PagedAttensionExtension expects 13 inputs, but it has ",
+                          get_input_size() == 15,
+                          "PagedAttensionExtension expects 15 inputs, but it has ",
```

Comment:
Doesn't look as optional inputs. According to the spec they could be omitted. If you replace by `get_input_size() == 13 || get_input_size() == 15` it wouldn't be a big code modification but unlock a bit of flexibility in the transition period where various mixes of main ov and genai may happen. As we keep PA op internal and not very particular on op version numbering, then a bit of backward compatibility care would be nice.